docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md

```diff
@@ -1018,7 +1105,100 @@ Advanced connector configuration properties:
+| MaxQueueSizeInBytes | `long` | The maximum buffer of the queue in bytes. This metric is available if `max.queue.size.in.bytes` is set to a positive long value. |
+| CurrentQueueSizeInBytes | `long` | The current volume, in bytes, of records in the queue. |
+
+### Streaming metrics
+
+The **MBean** is `debezium.yugabytedb:type=connector-metrics,server=<database.server.name>,task=<task.id>,context=streaming`.
+
+The following table lists the streaming metrics that are available.
+
+| Metric name | Type | Description |
+| :---- | :---- | :---- |
+| LastEvent | `string` | The last streaming event that the connector has read. |
+| MilliSecondsSinceLastEvent | `long` | The number of milliseconds since the connector has read and processed the most recent event. |
+| TotalNumberOfEventsSeen | `long` | The total number of events that this connector has seen since the last start or metrics reset. |
+| TotalNumberOfCreateEventsSeen | `long` | The total number of create events that this connector has seen since the last start or metrics reset. |
+| TotalNumberOfUpdateEventsSeen | `long` |The total number of update events that this connector has seen since the last start or metrics reset. |
+| TotalNumberOfDeleteEventsSeen | `long` | The total number of delete events that this connector has seen since the last start or metrics reset. |
+| NumberOfEventsFiltered | `long` | The number of events that have been filtered by include/exclude list filtering rules configured on the connector. |
+| QueueTotalCapacity | `int` | The length the queue used to pass events between the streamer and the main Kafka Connect loop. |
+| QueueRemainingCapacity | `int` | The free capacity of the queue used to pass events between the streamer and the main Kafka Connect loop. |
+| Connected | `boolean` | Flag that denotes whether the connector is currently connected to the database server. |
+| MilliSecondsBehindSource | `long` | The number of milliseconds between the last change event’s timestamp and the connector processing it. The values will incoporate any differences between the clocks on the machines where the database server and the connector are running. |
+| SourceEventPosition | `Map<String, String>` | The coordinates of the last received event. |
+| LastTransactionId | `string` | Transaction identifier of the last processed transaction. |
+| MaxQueueSizeInBytes | `long` | The maximum buffer of the queue in bytes. This metric is available if `max.queue.size.in.bytes` is set to a positive long value. |
+| CurrentQueueSizeInBytes | `long` | The current volume, in bytes, of records in the queue. |
+
+## Usage examples
 
+For examples on how to configure CDC in different setup configurations, you can see our [list of examples on GitHub](https://github.com/yugabyte/cdc-examples) for usage with various sinks and configuration information.
```

Comment:
```suggestion
For examples of how to configure CDC in different setup configurations, see the [list of examples on GitHub](https://github.com/yugabyte/cdc-examples) for usage with various sinks and configuration information.

```

Reply:
Addressed the comment from @aishwarya24 on this line.